### PR TITLE
Disable patch and package install and monitoring for sl micro minions

### PR DESCRIPTION
**Pull request by maximenoel8**
## What does this PR change?

Disable patch and package install and monitoring for sl micro minions in BV smoke test.

## GUI diff

No difference.

- [x] **DONE**

## Documentation

- No documentation needed: only internal and user invisible changes

- [x] **DONE**

## Test coverage
ℹ️ If a major new functionality is added, it is **strongly recommended** that tests for the new functionality are added to the Cucumber test suite
- No tests: already covered

- [x] **DONE**

## Links

- [x] **DONE**

## Changelogs

Make sure the changelogs entries you are adding are compliant with https://github.com/uyuni-project/uyuni/wiki/Contributing#changelogs and https://github.com/uyuni-project/uyuni/wiki/Contributing#uyuni-projectuyuni-repository

If you don't need a changelog check, please mark this checkbox:

- [x] No changelog needed

If you uncheck the checkbox after the PR is created, you will need to re-run `changelog_test` (see below)

## Re-run a test

If you need to re-run a test, please mark the related checkbox, it will be unchecked automatically once it has re-run:

- [ ] Re-run test "changelog_test"
- [ ] Re-run test "backend_unittests_pgsql"
- [ ] Re-run test "java_pgsql_tests"
- [ ] Re-run test "schema_migration_test_pgsql"
- [ ] Re-run test "susemanager_unittests"
- [ ] Re-run test "javascript_lint"
- [ ] Re-run test "spacecmd_unittests"

# Before you merge

Check [How to branch and merge properly](https://github.com/uyuni-project/uyuni/wiki/How-to-branch-and-merge-properly)!

### CODE REVIEW

**Review by Bischoff — APPROVED** [review]
ok

too bad we don't have bscs to reference for the other problems than prometheus?

**Comment by srbarrios** [comment]
Could we please explain why installation of packages and applying patches is disabled?

**Review by srbarrios — CHANGES_REQUESTED** [review]
Could we please discuss it in Slack before merging? Thank you. 🙏🏻

**Review by Bischoff — CHANGES_REQUESTED** [review]
It could be the situation has changed , see latest remarks from Abid about the exporter

Please do not merge yet, we need to discuss

**Comment by srbarrios** [comment]
Related to this, our images configured through cloud-init, should have now packages and patches available after the deployment, thanks to this refactor https://github.com/uyuni-project/sumaform/pull/1702

File: testsuite/features/build_validation/smoke_tests/smoke_tests.template
Comment (srbarrios) [inline testsuite/features/build_validation/smoke_tests/smoke_tests.template:180]: As discussed in Slack, I would personally show a red in this test case on the test report, unless this affects other test cases.
Otherwise, the test report is not reflecting the current product state.

File: testsuite/features/build_validation/smoke_tests/smoke_tests.template
Comment (srbarrios) [inline testsuite/features/build_validation/smoke_tests/smoke_tests.template:227]: Same.

File: testsuite/features/build_validation/smoke_tests/smoke_tests.template
Comment (srbarrios) [inline testsuite/features/build_validation/smoke_tests/smoke_tests.template:233]: Same

File: testsuite/features/build_validation/smoke_tests/smoke_tests.template
Comment (srbarrios) [inline testsuite/features/build_validation/smoke_tests/smoke_tests.template:241]: The skip looks good to me, but that's unrelated to the commented bugs.
Apache and Postgres components are not meant to be installed directly on SL Micro, so that's the reason why we don't support these exporters in the product, that's unrelated to Node Exporter which just collects HW metrics.

File: testsuite/features/build_validation/smoke_tests/smoke_tests.template
Comment (srbarrios) [inline testsuite/features/build_validation/smoke_tests/smoke_tests.template:59]: We should not be skipping them, this is a supported feature.
Instead, we need to find a way to cover this with a patch, if none available now.

File: testsuite/features/build_validation/smoke_tests/smoke_tests.template
Comment (srbarrios) [inline testsuite/features/build_validation/smoke_tests/smoke_tests.template:74]: We should not be skipping them, this is a supported feature for SL/SLE Micro.
Instead, we need to find a way to cover this with a package, if none available now.

Hopefully this PR will help on this https://github.com/uyuni-project/sumaform/pull/1702

**Pull request closed without merging**
